keps/sig-autoscaling/5325-hpa-pod-selection-accuracy/README.md

```diff
@@ -0,0 +1,1024 @@
+
+###### What specific metrics should inform a rollback?
+
+<!--
+What signals should users be paying attention to when the feature is young
+that might indicate a serious problem?
+-->
+Operators should monitor these signals that might indicate problems:
+
+- Unexpected scaling events shortly after enabling the feature
+- Significant changes in the number of replicas for workloads using HPAs with `selectionStrategy: OwnerReference`
+- Increased latency in the `horizontal_pod_autoscaler_controller_metric_computation_duration_seconds` metric
+- Increased error rate in `horizontal_pod_autoscaler_controller_metric_computation_total` with error status
+If these metrics show unusual patterns after enabling the feature, operators should consider rolling back.
+
+###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
+
+<!--
+Describe manual testing that was done and the outcomes.
+Longer term, we may want to require automated upgrade/rollback tests, but we
+are missing a bunch of machinery and tooling and can't do that now.
+-->
+
+###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
+
+<!--
+Even if applying deprecation policies, they may still surprise some users.
+-->
+No. This feature only adds a new optional field to the HPA API and doesn't deprecate or remove any existing functionality.
+All current HPA behaviors remain unchanged unless users explicitly opt into the new selection mode.
+
+### Monitoring Requirements
+
+<!--
+This section must be completed when targeting beta to a release.
+
+For GA, this section is required: approvers should be able to confirm the
+previous answers based on experience in the field.
+-->
+
+###### How can an operator determine if the feature is in use by workloads?
+
+<!--
+Ideally, this should be a metric. Operations against the Kubernetes API (e.g.,
+checking if there are objects with field X set) may be a last resort. Avoid
+logs or events for this purpose.
+-->
+The presence of the `selectionStrategy` field in HPA specifications indicates that the feature is in use.
+
+###### How can someone using this feature know that it is working for their instance?
+
+<!--
+For instance, if this is a pod-related feature, it should be possible to determine if the feature is functioning properly
+for each individual pod.
+Pick one more of these and delete the rest.
+Please describe all items visible to end users below with sufficient detail so that they can verify correct enablement
+and operation of this feature.
+Recall that end users cannot usually observe component logs or access metrics.
+-->
+
+Users can confirm that the feature is active and functioning by inspecting the conditions exposed by the controller. Specifically, they can verify the value of `.spec.selectionStrategy` to ensure the expected behavior is enabled.
```

Comment:
This tells users that it's being specified, but not that it's necessarily working - the HPA event for switching selection strategy is maybe more useful?